src/evm-block-extractor/src/database/postgres_db_client.rs

```diff
@@ -237,6 +237,79 @@ impl DatabaseClient for PostgresDbClient {
             .map_err(|e| anyhow::anyhow!("Error getting transaction {}: {:?}", hex_tx_hash, e))
             .and_then(|row| from_row_value(&row, 0))
     }
+
+    async fn discard_tail(&self, start_from: u64, reason: &str) -> anyhow::Result<()> {
```

Comment:
Add proper logging. Dropping blocks should be at least at a `warn` level

Reply:
Done